src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { selectFromStore } from '../../store/selectors/pokemon.selectors';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnInit {
+  private url = "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/"
```

Comment:
the url for the images must be a environment variable located in the environment.ts